.github/workflows/ci.yml

```diff
@@ -1,5 +1,8 @@
 name: Continuous Integration
 on: [push, pull_request]
+env:
+  GITHUB_BASE: origin/${{ github.event.pull_request.base.ref }}
```

Comment:
We can add another variable that would hold `github.event.before`, and use that in case the event is a sole `push`.

Reply:
`event.before` often refers to the previously-CI'ed commit AFAICT and isn't what we wanted

Reply:
Where can I find documentation for these fields?  It looked like `event.before` was not actually what we wanted

Reply:
I believe `github.event.before` is not the target branch ref, but rather the literal previous thing that was tested via GHA (e.g.the previous commit)